Quickstore/pages/signin.html

```diff
@@ -12,25 +12,34 @@
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
      
      <!---------Style sheet of home page------>
-     <link rel="stylesheet" href="/Quickstore/pages-styles/signin.css">
+     <link rel="stylesheet" href="/Quickstore/pages-styles/quickshop.css">
 </head>
 <body>
     <!-----------Navbar---------->
     <nav class="nav-heading">
-        <h1 class="branch-heading">QuickShop</h1>
-        <div class="search-bar">
-            <input type="text" class="search-input-box"  >
-            <div class="search-button">
-            <a href="#"><i class="search fa fa-search"></i> </a>
-           </div>
-        </div>
-        <div class="login-section">
-            <a href="#"><i class="social-icon fa fa-user"></i><p>Login</p></a>
-            <a href="#"><i class="social-icon fa fa-heart"></i><p>Wishlist</p></a>
-            <a href="#"><i class="social-icon fa fa-shopping-cart"></i><p>Cart</p></a>
-        </div>
+        <h1 class="home-heading">QuickStore</h1> 
+            <div class="search-bar">
+                <input type="text" class="search-input-box" placeholder=" Seach for latest">
+                <div class="search-button">
+                    <a href="#"><i class="fa fa-search"></i></a>
+                </div>
+            </div>
+            <div class="login-section">
+                    <a href="/Quickstore/pages/signin.html"><i class="badge-icon fa fa-user"></i><p>Login</p></a>
+                    <div class="wishlist-badge badge">
+                        <a href="/Quickstore/pages/wishlist.html"> <i class="badge-icon fa fa-heart"></i> 
+                            <p class="names">wishlist</p>
+                            <span class="badge-symbol">3</span></a>
```

Comment:
suggestion : badge-symbol makes less sense to the code, instead you can use badge-content

Reply:
if i change it will become complicated i have also change in component libaray